src/styles/index.css

```diff
@@ -0,0 +1,4 @@
+@import "../styles/card.css";
+@import "../styles/reset.css";
+@import "../styles/stars.css";
+@import "../styles/variables.css";
```

Comment:
move variables import to the top of file

Reply:
**Fixed**